src/pages/trader-tools/_trading-tools.tsx

```diff
@@ -56,7 +56,7 @@ const Column = styled.div<ColumnProps>`
         flex-direction: column;
     }
 `
-const Content = styled(Flex)`
+const Content = styled(Flex)<{ margin_left?: string; margin_right?: string }>`
     max-width: 39rem;
     margin-right: ${(props) => props.margin_right};
```

Comment:
since the margin_left and margin_right props are optional we need to make sure that if they had those props have values we add those attributes.

Reply:
@ali-hosseini-fs I checked that, we are using these properties, I just need to remove optional chain from them because they are required